go.mod

```diff
@@ -3,7 +3,10 @@ module github.com/mattermost/mattermost-plugin-starter-template
 go 1.12
 
 require (
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
Should these changes be in `build/go.mod`?

Reply:
@tasdomas What's your opinion on this?